renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>NethServer/.github:renovate-config"
```

Comment:
looking to [mattermost](https://github.com/NethServer/ns8-mattermost/blob/main/renovate.json), I am not sure it is good, we have a lot of PR to the UI

can we do like mattermost : `"local>NethServer/.github:ns8"`

Reply:
Yes, try to copy Mattermost.
We are going to use the same config for all repos in the near future.

Reply:
done